FreeRTOS-Plus/Demo/AWS/Mqtt_Demo_Helpers/mqtt_demo_helpers.c

```diff
@@ -119,7 +119,7 @@
 /**
  * @brief Timeout for MQTT_ProcessLoop in milliseconds.
  */
-#define mqttexamplePROCESS_LOOP_TIMEOUT_MS           ( 500U )
+#define mqttexamplePROCESS_LOOP_TIMEOUT_MS           ( 2000U )
```

Comment:
How did we arrive at 2secs number. I understand that the previous value of 500ms was less. 
Ideally there should be a formula which co-relates the publish counts and mqttexamplePROCESS_LOOP_TIMEOUT_MS. And when we make the publish count as 15, we should get the above value.

Reply:
The arrival on 2sec number is based on periodically increasing mqttexamplePROCESS_LOOP_TIMEOUT_MS 
 and at what timeout we are able to keep up with the incoming ACKs and publishes.

Reply:
Please add this reasoning. After experimenting it has been found that for 15 publish messages, the apt timeout value is 2 secs. This should be updated if the number of publish messages change.

Reply:
Updated